packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -603,7 +604,7 @@ export class ContextualMenuBase extends BaseComponent<IContextualMenuProps, ICon
     const { itemProps } = item;
 
     return (
-      <div className={this._classNames.header} style={item.style}>
+      <div className={this._classNames.header} style={item.style} {...getNativeProps(item, divProperties)}>
```

Comment:
For this to work and be type-safe, _I think_ we would want to call `getNativeProps(itemProps, divProperties)` since `itemProps` is of type `IContextualMenuItemProps` which extends `React.HTMLAttributes`.

https://github.com/OfficeDev/office-ui-fabric-react/blob/bfc8ca5b7c17a54530ae51b46d57fb9546e6e861/packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItem.types.ts#L24

Reply:
Also, the object spread we're introducing here may include a `style` attribute which might override `style={item.style}`... unsure at a glance. It would be good to add some tests if we decide to implement this.

Reply:
I move it to ItemProps.  I also noticed that there was a header demo page but it wasn't added to the Contextual Menu page, so I added it there and set the ItemProps on one of the headers to verify.